saleor/csv/emails.py

```diff
@@ -21,7 +21,10 @@
 def send_email_with_link_to_download_file(
     export_file: "ExportFile", template_name: str
 ):
-    recipient_email = export_file.created_by.email
+    user = export_file.user
+    if not user:
+        return
```

Comment:
In what cases there will be no user? When an app is requesting the export? How would we provide the export results then?

Reply:
I talked with @korycins about that. The app can query `ExportFile` with the use of ExportFile ID returned by mutation (current solution). Another possible solution, which can be implemented, is adding an optional `email` field to `exportProducts` mutation.

Reply:
OK, but I think these functions are expected to send the email, and these checks should be moved outside, to where these tasks are actually called. Same for the other email sending function where we added this check.

Reply:
I did it in this way first (as you proposed), but then I moved it here because it will fail when user in export_file will be None. But maybe I should add the `user` parameter to force providing it.

Reply:
Adding explicit `user` param sounds good to me, I think we do that in other email functions, or we explicitly pass email as string even. It's not very consistent, unfortunately.